web calculator/web calculator/index.html

```diff
@@ -0,0 +1,20 @@
+<html>
+<h1>CALCULATOR</h1>
+    
+    <head>
+        <script src="script.js"></script>
```

Comment:
While you loaded your Javacript file, you never called your function from your HTML. You also don't have a space for your answer to live in. You could have used an empty HTML element like a span or even the alert function.